### PR TITLE
Fixes path for openapi-generator output in README.md

### PR DESCRIPTION
**Pull request by trebler**
Generated code is under _/src/gen_, not just _/src_

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -119,7 +119,7 @@ npm install
 cd ../
 git clone https://github.com/kubernetes-client/gen
 cd javascript
-../gen/openapi/typescript.sh src settings
+../gen/openapi/typescript.sh src/gen settings
```
Comment (drubin) [inline README.md]: @silasbw does one need the `USERNAME` param?

`USERNAME=kubernetes ../gen/openapi/typescript.sh src/gen settings`

From https://github.com/kubernetes-client/javascript/pull/307

File: README.md [hunk 2]
```diff
@@ -119,7 +119,7 @@ npm install
 cd ../
 git clone https://github.com/kubernetes-client/gen
```
Comment (drubin) [inline README.md]: Totally separate from this PR, but @silasbw @brendanburns should we not be version locking this so it's reproducible?

Maybe add a CI check to verify that the generated files match the version in "code"

Comment (drubin) [reply README.md]: I am going to resolve this in favour of an issue #325 325

Comment (silasbw) [reply README.md]: This makes sense to me.

File: README.md (hunk 1 above)
Comment (silasbw) [reply README.md]: I think so: that ensures we clone from the kubernetes org (and not, for example, someone's fork).

**Comment by drubin** [comment]
/lgtm
/assign @brendandburns 

We can fix the issues in other PR's because they are a bunch more work

**Pull request merged**